settings.gradle.kts

```diff
@@ -1,16 +1,25 @@
-rootProject.name = "multiplatform-paging-root"
-
 enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
 
 pluginManagement {
   repositories {
-    gradlePluginPortal()
     google()
+    gradlePluginPortal()
     mavenCentral()
+    maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
```

Comment:
Similar to above, would prefer the order be plugin portal, maven central, google. We shouldn't need the JB Compose dev repo.